spec/helpers/admin_only/dashboard_helper_spec.rb

```diff
@@ -220,7 +220,7 @@
           expect(helper.styled_item_then_text('css-class',
                                               'leading item',
                                               'some text',
-                                              spacer: '--')).to match /<span class=\"css-class\">leading item<\/span>(.)+some text/
+                                              spacer: '--')).to match(/<span class=\"css-class\">leading item<\/span>(.)+some text/)
```

Comment:
revert this change and put it into the `.rubocop_todo.yml` file

Reply:
For some reason this is the only offense that is not ignored if I put it into the .rubocop_todo.yml file. That's why I kept the auto-correction.